app/lib/frontend/backend.dart

```diff
@@ -159,6 +162,119 @@ class Backend {
     assert(repository.supportsDownloadUrl);
     return repository.downloadUrl(package, version);
   }
+
+  /// Stores a [models.PackageInvite] entry in the Datastore and returns its
+  /// urlNonce can be used in client-communication, e.g. sending via e-mail.
+  ///
+  /// Returns null if invite has been updated recently and at the moment no
```

Comment:
Maybe null is not enough information? Should we not send a message back to the client to display?

Reply:
I do not have enough overview of the program structure to answer that. I just think it might be nice to answer the client with why no new mail has been sent, and when the wait is over.

Reply:
PTAL. I've added a return value class that has enough information to communicate both the success or the amount of wait.